sql/core/src/test/scala/org/apache/spark/sql/test/SharedSparkSession.scala

```diff
@@ -72,6 +72,8 @@ trait SharedSparkSessionBase
       // this rule may potentially block testing of other optimization rules such as
       // ConstantPropagation etc.
       .set(SQLConf.OPTIMIZER_EXCLUDED_RULES.key, ConvertToLocalRelation.ruleName)
+      // Reduce shuffle partitions to reduce testing time.
+      .set(SQLConf.SHUFFLE_PARTITIONS.key, "5")
```

Comment:
I think you can remove here:

https://github.com/apache/spark/blob/ec8a1a8e8869b000a8001126ca013b753acddf6a/sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala#L297

Reply:
Done